cmd/prom-migrator/main.go

```diff
@@ -48,6 +49,11 @@ type config struct {
 
 func main() {
 	conf := new(config)
+    args := os.Args[1:]
+    if shouldProceed := runner.ParseArgs(args); !shouldProceed {
+        os.Exit(0)
+    }
```

Comment:
Importing this from the runner package causes the problem. This might be true in itself because the runner package was aimed to be specific to Promscale only.

This can be solved by making a `parseArgs()` function in `cmd/prom-migrator/main.go`, which is kind of a replica of runner.ParseArgs(). Like,

```golang
func parseArgs(args []string) (shouldProceed bool) {
	shouldProceed = true // Some flags like 'version' are just to get information and not proceed the actual execution. We should stop in such cases.
	for _, flag := range args {
		flag = flag[1:]
		switch flag {
		case "version":
			shouldProceed = false
			fmt.Println(version.VersionPromMigrator)
		}
	}
	return
}
```

And then in `pkg/version/version.go`, we require a version for Prom-migrator that should be printed here. You can add a `VersionPromMigrator` and import that in `main.go` as explained above and that should be enough.

Also, I think it will be better to convert `Version` to `VersionPromscale` and make the related name changes across the repo. That should be good to go.

Reply:
Lets change `VersionPromscale` => `Promscale` and `VersionPromMigrator` => `PromMigrator` so that while importing, it becomes `version.Promscale` and `version.PromMigrator` which is self-explanatory.

Reply:
Wow thanks for the detailed steps @Harkishen-Singh, I just did the proposed changes and `git rebase master` before pushing, let me know if there is something else we need for merging this change.